### ISSUE TITLE
libswiftCore.dylib`swift_unknownUnownedLoadStrong:

### ISSUE THREAD

**Issue by jassimkm** [AssistoLab/DropDown#45]
```
    My App is crashing here line 485 in DropDown.swift . I cant figure out why . I am not using DropDown in viewcontroller it is crashing .

    dispatch_async(dispatch_get_main_queue()) { [unowned self] in
        self.tableView.flashScrollIndicators()
    }
    super.updateConstraints()
```

**Comment by zanesc** [comment]
I am having this same issue. did you figure it out?

**Comment by kevin-hirsch** [comment]
This problem is probably caused because you do not retain the drop down in a variable outside of a scope that ends just after the call to `show()`. 

You are probably doing this:
```swift
func someMethod() {
   let dropDown = DropDown() // var declared + instantiated
   // dropDown setup...
   dropDown.show()
} // dropDown var is released
```

The `dropDown` var is actually not retained by any view (its `anchorView` does not retain it), so it is released at the end of the scope.

What you have to do is retain it somewhere like this for example:

```swift
class MyVC: UIViewController {
   let dropDown = DropDown() // class instance var, is now retained by its instance

   override open viewDidLoad() {
      super.viewDidLoad()
      // dropDown setup...
   }

   func someMethod() {
      dropDown.show()
   }
}
```

Can you confirm it solves the problem? :)

**Comment by carlosen14** [comment]
Actually I am creating and storing them in an `Array [DropDown]`.


`  
//at the global class variables
var arrDropDown: [DropDown] = []

//before anything fill the array
      self.arrDropDown = [DropDown](repeating: DropDown(), count: self.data.count)

//inside function

      let drop = DropDown()
      self.arrDropDown.insert(drop, at: indexPath.row)

     //also doing
     self.arrDropDown[indexPath.row] = drop
`

**Comment by carlosen14** [comment]
I just commented the line 
`self.tableView.flashScrollIndicators()`
and the exception is gone, but now I get an advice
`[LayoutConstraints] Unable to simultaneously satisfy constraints.`

**Comment by piterwilson** [comment]
Same issue here

**Comment by kevin-hirsch** [comment]
Hello!
I've pushed a new commit that should fix this problem (80a125b on master)
